tests/entrypoints/llm/test_lazy_outlines.py

```diff
@@ -29,7 +29,7 @@ def test_lazy_outlines(sample_regex):
     llm = LLM(model="facebook/opt-125m",
               enforce_eager=True,
               guided_decoding_backend="lm-format-enforcer",
-              gpu_memory_utilization=0.3)
+              gpu_memory_utilization=0.6)
```

Comment:
I think this test was working before due to the over-estimation of peak memory usage of the model which caused a smaller KV cache to be allocated. Two `LLM`s both set `gpu_memory_utilization=0.3`, but once the first LLM uses the full 30% of the gpu, there's no space left to allocate room for the second one.

This setting is a bit confusing- how it has been coded is "The total GPU allocation may not exceed x% of the gpu memory when loading this model", but it looks like the test assumed the setting meant "You may not allocate more than x% of the gpu memory for this model, regardless of how much of the gpu memory ends up being allocated." In other words, it assumed this was a per-model limit and not a global limit on gpu memory allocation.

Maybe that should be made more clear in the docs?

Reply:
Thanks for the explanation. Could you add a short comment, since the reader may find it odd that the second call sets `gpu_memory_utilization` differently from the first?

Alternatively, looks like the first llm doesn't need to be live when the second one is created, so we could try to force it to be garbage collected but I don't think it's worth jumping through hoops for this

Reply:
> I don't think it's worth jumping through hoops for this

I agree :D 

I added a small comment here for future readers